### DISCUSSION TITLE
Is public channel like discord, github or gitter available for question/discussion?

### DISCUSSION THREAD

**Discussion by MohitGupta121** [metabrainz/musicbrainz-android#135] [General]
Or in musicbrainz using GitHub discussion section for that purpose?

**Comment by amCap1712** [comment]
Hi! You can reach out to us in the #metabrainz IRC Channel. Instructions to join are at [https://wiki.musicbrainz.org/Communication/IRC](https://wiki.musicbrainz.org/Communication/IRC)

**Reply by MohitGupta121** [reply]
Okay, Thank you so much @amCap1712 Buddy.
